### PR TITLE
make it possible to skip popover

### LINKED ISSUE #597: Make it possible to skip popover
there should be an option to skip the popover and always open the sidebar for editing.

Default setting should be to open the popover first.

### PR DESCRIPTION
**Pull request by georgehrke**
fixes https://github.com/owncloud/calendar/issues/597

### CODE REVIEW

**Comment by georgehrke** [comment]
please review @raghunayyar

File: controller/settingscontroller.php [hunk 1]
```diff
@@ -51,18 +56,23 @@ public function __construct($appName, IRequest $request, IUserSession $userSessi
 		parent::__construct($appName, $request);
 		$this->config = $config;
 		$this->userSession = $userSession;
+		$this->userId = $userSession->getUser()->getUID();
 	}
 
 	/**
 	 * get a configuration item
 	 *
+	 * @NoAdminRequired
```
Comment (tcitworld) [inline controller/settingscontroller.php:65]: Oops.

**Comment by tcitworld** [comment]
Looks fine to me.

**Comment by georgehrke** [comment]
thx for testing :)

**Pull request merged**